modules/master-node-sizing.adoc

```diff
@@ -109,7 +109,12 @@ Operator Lifecycle Manager (OLM ) runs on the control plane nodes and it's memor
 
 [IMPORTANT]
 ====
-If you used an installer-provisioned infrastructure installation method, you cannot modify the control plane node size in a running {product-title} {product-version} cluster. Instead, you must estimate your total node count and use the suggested control plane node size during installation.
+You can modify the control plane node size in a running {product-title} {product-version} cluster for the following configurations only:
+
+* Clusters installed with a user-provisioned installation method.
+* AWS clusters installed with an installer-provisioned infrastructure installation method.
+
```

Comment:
@sheriff-rh I am not sure if this part only include these two situations, I checked with @geliu2016  who did master replacement test, we also tested the master replacement for ipi vsphere.

Reply:
What about line 117:
"For all other configurations, you must estimate your total node count and use the suggested control plane node size during installation."

Does that accurately cover all situations?

Reply:
Sound good to me, thanks @sheriff-rh